lib/vdsm/storage/volume.py

```diff
@@ -1634,6 +1637,7 @@ class VolumeLease(guarded.AbstractLock):
     """
     Extend AbstractLock so Volume Leases may be used with guarded utilities.
     """
+
```

Comment:
Unrelated change, best to avoid such changes.

If you want to unify spacing in the entire code base do this separately.

Reply:
`autopep8` trolled me a bit on this one, I will be more careful with it, sorry.